lib/plugin/utils/plugin-utils.ts

```diff
@@ -331,6 +334,24 @@ function isOptionalBoolean(text: string) {
   return typeof text === 'string' && text === 'boolean | undefined';
 }
 
+function adaptPathWithConfig(inputPath: string) {
+  try {
+    const rawConfig = readFileSync('tsconfig.json', 'utf8');
```

Comment:
I'm concerned this hard code `"tsconfig.json"`

As a dev, we can name our tsconfig file as we wish, so this readFile will always fail.

Not sure how to handle this properly